buildbot/master/files/config/master.cfg

```diff
@@ -8,10 +8,11 @@ from passwords import SLAVE_PASSWORD, CHANGE_PASSWORD
 from passwords import HOMU_BUILDBOT_SECRET, GITHUB_STATUS_TOKEN
 
 
-LINUX_SLAVES = ["servo-linux1", "servo-linux2", "servo-linux3"]
+LINUX_SLAVES = ["servo-linux1", "servo-linux2", "servo-linux3", "servo-linux4",
```

Comment:
Yeah just a style comment, this would look nicer to me:
```
LINUX_SLAVES = [
    "servo-linux1",
    "servo-linux2",
    ...
    "servo-linux5",
    "servo-linux6,
]
```
or doing that with multiple names per line,
or even possibly
```
LINUX_SLAVES = ["servo-linux{}".format(i) for i in xrange(1, 7)]
```